### ISSUE TITLE
Confirm SSL certificate doesn't blow up on May 30

### ISSUE THREAD

**Issue by colinxfleming** [DCAFEngineering/dcaf_case_management#1041]
Thanks for creating an issue! Please fill out this form so we can be sure to have all the information we need, and to minimize back and forth.

* **What are we trying to do?**
LetsEncrypt certs have an expiration period of 90 days, and the first one of those deadlines since we rolled them up is coming up on <strike>May 14</strike> May 30 at 22:37 GMT. After that benchmark we should confirm our LetsEncrypt rig is still good to go.

Initial perusal indicates an expiration date of July 30 but better to be safe, because production system, etc. 

* **What feature or behavior is this required for?**
~ security ~ 

* **How could we solve this issue? (Not knowing is okay!)**
On <strike>May 14</strike> in the late evening, confirm that our SSL certificates are still good to go and haven't expired.

* **Anything else?**
Nope!

**Comment by colinxfleming** [comment]
14th came and went without incident; sandbox reads July 30, prod reads may 30. Should check.

**Comment by colinxfleming** [comment]
It blew up due to the rake task failing silently. see result:

```
$ heroku run -a PROD 'rake letsencrypt:renew'
Running rake letsencrypt:renew on ⬢ PROD... up, run.4573 (Hobby)
Creating account key...Done!
Registering with LetsEncrypt...Done!
Using ACME_DOMAIN configuration variable...
Performing verification for PROD:
Setting config vars on Heroku...rake aborted!
Excon::Error::Unauthorized: Expected([200, 201, 202, 204, 206, 304]) <=> Actual(401 Unauthorized)
/app/vendor/bundle/ruby/2.4.0/gems/excon-0.55.0/lib/excon/middlewares/expects.rb:7:in `response_call'
/app/vendor/bundle/ruby/2.4.0/gems/excon-0.55.0/lib/excon/middlewares/response_parser.rb:9:in `response_call'
/app/vendor/bundle/ruby/2.4.0/gems/excon-0.55.0/lib/excon/connection.rb:388:in `response'
/app/vendor/bundle/ruby/2.4.0/gems/excon-0.55.0/lib/excon/connection.rb:252:in `request'
/app/vendor/bundle/ruby/2.4.0/gems/heroics-0.0.21/lib/heroics/link.rb:111:in `request_with_cache'
/app/vendor/bundle/ruby/2.4.0/gems/heroics-0.0.21/lib/heroics/link.rb:66:in `run'
/app/vendor/bundle/ruby/2.4.0/gems/heroics-0.0.21/lib/heroics/resource.rb:28:in `method_missing'
/app/vendor/bundle/ruby/2.4.0/bundler/gems/platform-api-4a06e57bd32d/lib/platform-api/client.rb:1131:in `update'
/app/vendor/bundle/ruby/2.4.0/gems/letsencrypt-rails-heroku-1.2.0/lib/tasks/letsencrypt.rake:46:in `block (3 levels) in <top (required)>'
/app/vendor/bundle/ruby/2.4.0/gems/letsencrypt-rails-heroku-1.2.0/lib/tasks/letsencrypt.rake:39:in `each'
/app/vendor/bundle/ruby/2.4.0/gems/letsencrypt-rails-heroku-1.2.0/lib/tasks/letsencrypt.rake:39:in `block (2 levels) in <top (required)>'
/app/vendor/bundle/ruby/2.4.0/gems/rake-12.0.0/exe/rake:27:in `<top (required)>'
/app/vendor/bundle/ruby/2.4.0/gems/bundler-1.13.7/lib/bundler/cli/exec.rb:74:in `load'
/app/vendor/bundle/ruby/2.4.0/gems/bundler-1.13.7/lib/bundler/cli/exec.rb:74:in `kernel_load'
/app/vendor/bundle/ruby/2.4.0/gems/bundler-1.13.7/lib/bundler/cli/exec.rb:27:in `run'
/app/vendor/bundle/ruby/2.4.0/gems/bundler-1.13.7/lib/bundler/cli.rb:332:in `exec'
/app/vendor/bundle/ruby/2.4.0/gems/bundler-1.13.7/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
/app/vendor/bundle/ruby/2.4.0/gems/bundler-1.13.7/lib/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
/app/vendor/bundle/ruby/2.4.0/gems/bundler-1.13.7/lib/bundler/vendor/thor/lib/thor.rb:359:in `dispatch'
/app/vendor/bundle/ruby/2.4.0/gems/bundler-1.13.7/lib/bundler/cli.rb:20:in `dispatch'
/app/vendor/bundle/ruby/2.4.0/gems/bundler-1.13.7/lib/bundler/vendor/thor/lib/thor/base.rb:440:in `start'
/app/vendor/bundle/ruby/2.4.0/gems/bundler-1.13.7/lib/bundler/cli.rb:11:in `start'
/app/vendor/bundle/ruby/2.4.0/gems/bundler-1.13.7/exe/bundle:34:in `block in <top (required)>'
/app/vendor/bundle/ruby/2.4.0/gems/bundler-1.13.7/lib/bundler/friendly_errors.rb:100:in `with_friendly_errors'
/app/vendor/bundle/ruby/2.4.0/gems/bundler-1.13.7/exe/bundle:26:in `<top (required)>'
/app/bin/bundle:3:in `load'
/app/bin/bundle:3:in `<main>'
Tasks: TOP => letsencrypt:renew
(See full trace by running task with --trace)
```

I'm going to see if moving it to the ACM is trivial.

**Comment by colinxfleming** [comment]
Resolved via #1137
